pkg/reconciler/brokercell/controller.go

```diff
@@ -23,6 +23,12 @@ import (
 
 	brokercell "github.com/google/knative-gcp/pkg/client/injection/informers/intevents/v1alpha1/brokercell"
 	v1alpha1brokercell "github.com/google/knative-gcp/pkg/client/injection/reconciler/intevents/v1alpha1/brokercell"
+	"github.com/kelseyhightower/envconfig"
+	"go.uber.org/zap"
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+	deploymentinformer "knative.dev/pkg/client/injection/kube/informers/apps/v1/deployment"
+	endpointsinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/endpoints"
+	serviceinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/service"
 	configmap "knative.dev/pkg/configmap"
 	controller "knative.dev/pkg/controller"
 	logging "knative.dev/pkg/logging"
```

Comment:
I recommend using [the logging package from eventing](https://github.com/knative/eventing/blob/master/pkg/logging/logging.go). It uses desugared loggers for better safety.

Also I wrote this code and it was wrong 😅